Av1an/bar.py

```diff
@@ -64,7 +62,7 @@ def process_pipe(pipe):
 
 
 def make_vvc_pipe(command):
-    pipe = subprocess.Popen(command.split(), stdout=PIPE,
+    pipe = subprocess.Popen(command[0], stdout=PIPE,
```

Comment:
`command` here was only 1 string and is now only 1 tuple, so no indexing is needed.

```suggestion
    pipe = subprocess.Popen(command, stdout=PIPE,
```

Reply:
compose_vvc creates a Tuple[Tuple[str]], like the other compose functions. I did this to be consistent with the other compose functions and to be prepared if we want to pipe from ffmpeg later. I do not know about vvc, but I guess it does not support reading from pipe (yet?). If you prefer using a Tuple[str] you also need to change compose_vvc.

Reply:
Sorry, you were right. I intended it to be a `Tuple[Tuple[str]]`, but looks like I missed a pair of brackets. It is all right with your change.